test/unit/files/sentry_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative File.expand_path("../../spec_helper.rb", __FILE__)
+require_relative File.expand_path("../../../../files/default/sentry.rb", __FILE__)
+
+require 'chef'
+
+describe Raven::Chef::SentryHandler do
+
+  before do
+    @node = Chef::Node.new
+    @node.chef_environment = "testing"
+    @node.default["sentry"] = {
+      "dsn" => "http://test.test",
+      "verify_ssl" => true
+    }
+  end
+
+  describe "#sanitize_exception" do
+    context "exception has a keyword" do
+      it "sanitizes with a keyword" do
+        exception = "there's a key here"
+
+        expect(described_class.new(@node).send(:sanitize_exception, exception)).to eq(SANITIZED_EXCEPTION_MESSAGE)
```

Comment:
normally it would just be 
`described_class.new(@node).sanitize_exception(exception)`
but `send` allows you to access the private method.

I'm not 100% sure on the internals of this (going to look into it just to know), but yeah it seems weird